js/connection.js

```diff
@@ -289,7 +289,8 @@ weechat.factory('connection',
             if (timeout >= 600000) {
                 // If timeout is ten minutes or more, give up
                 $log.info('Failed to reconnect, giving up');
-                handleClose();
+                // This code can't run since handleClose is defined inside connect
+                // handleClose();
```

Comment:
Sure, but that doesn't have to be a part of this PR. And that code wouldn't be able to run anyway so I'm not changing anything really.